Rakefile

```diff
@@ -309,6 +308,7 @@ end
 serf_static_lib = File.join(serf_build_dir, 'libserf-1.a')
 file serf_static_lib => [installed_pkg_config, installed_openssl, installed_zlib, scons_build_dir, serf_build_dir] do
   xcode_sdk_root = `/usr/bin/xcrun --show-sdk-path`.chomp
+  scons_bin = File.expand_path(File.join(Dir[scons_build_dir + '/scons*'][0].to_s, '/src/script/scons.py'))
```

Comment:
I think this is a bad idea, when an update comes, a user may have the old version and a new version and this may not select the correct version and could lead to someone shipping a release with the wrong version of icons.

Perhaps we could error if `Dir[scons_build_dir + '/scons*']` returns > 1?